api.md

```diff
@@ -287,6 +288,12 @@ in the table.
 to how you handle constraints. A hypertable can contain foreign keys to normal SQL table
 columns, but the reverse is not allowed. UNIQUE and PRIMARY constraints must include the
 partitioning key.
+>
+>The deadlock is likely to happen when concurrent transactions simultaneously try 
+to insert data into tables that are referenced in the foreign key constraints, and into the
+converting table itself. The deadlock can be prevented by manually obtaining `SHARE ROW EXCLUSIVE` lock 
```

Comment:
Many nested and interleaved concepts making it hard to read, suggest something like this:

> If the table have foreign key constraints to other tables, a deadlock can happen if other transactions insert into the referenced table at the same time as data is migrated into the hypertable. The deadlock can be prevented ....

Reply:
IMHO starting sentence with `if` is not a good writing style. And your suggestion unfortunately is rather hard to read due to 2 ifs in it.
I will see if I can simplify the sentence.